### DISCUSSION TITLE
Is the delete file function broken in v5.18 beta?

### DISCUSSION THREAD

**Discussion by schulle4u** [BearWare/TeamTalk5#2482] [Q&A] [answered]
I am currently using Qt client v5.18.0.5150 (beta) on Windows, the server is still running version 5.17. It is currently not possible to delete a file on any of the servers I have tested, even when using an admin account. Neither the delete button nor the delete option in the context menu can be activated. There is no error message, but the client does not seem to react at all, as if it were a button with no function. Users of the stable client version 5.17 have no problems with this. Is this a known bug or is there something I can test?

**Comment by lbk2907** [comment]
I'm facing the same issue. As for now, the only way to delete the file is
to delete all files in the channel by pressing Control+A to select all the
files in that channel before you can delete it.

**Comment by schulle4u** [comment, marked as answer]
Meanwhile I have figured out how to delete individual files. Apparently there is some kind of checkbox in front of the file name, but my screen reader does not recognize it as such. For example, in NVDA, if you use the object navigation to jump from the file name to the left, an unnamed object called "Heading" appears. This can be activated with an emulated left mouse click (NVDA+Enter won't work in this case), after which you can press Shift+Delete as usual to delete the file. So apparently it is rather a kind of accessibility problem.

**Comment by lbk2907** [comment]
@schulle4u Thanks for the work around. At least this what we can do before the actual fix.
